packages/dialect-react-ui/components/Broadcast/BroadcastForm.tsx

```diff
@@ -120,13 +164,40 @@ function BroadcastForm({ dapp }: BroadcastFormProps) {
     }
   };
 
+  const renderNotificationTypeSelect = () => {
+    if (!notificationsSubscriptions.length) {
+      return '📢 Broadcast';
+    }
+
+    if (notificationsSubscriptions.length === 1) {
+      return notificationsSubscriptions[0]?.notificationType.name;
+    }
+
+    return (
+      // TODO: create a preflighted version of select with :focus-visible and other default things, which is already configured for inputs and buttons
+      <select
```

Comment:
I'd add a `value` prop to `select` as well so that his component is controlled.
```typescript
<select ... value={notificationTypeId}>...</select>
```